src/helpers/helpers.ts

```diff
@@ -22,7 +22,7 @@ export const customPageView = (url: string) => {
 
 export const customOutboundLink = (url: string) =>
   develop
-    ? Object.assign(document.createElement('a'), { target: '_blank', href: url, rel: 'noopener noreferrer'}).click()
+    ? window.open(url, '_blank')
```

Comment:
This solution has a vulnerability issue (https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/). I would suggest to revert this commit and to append the ancor element to the dom before clicking it (appending it to the dom should solve the previous issue of the page not opening)